xla/hlo/ir/hlo_instruction.cc

```diff
@@ -1008,9 +1008,16 @@ absl::StatusOr<std::unique_ptr<HloInstruction>> HloInstruction::CreateFromProto(
         for (const ShapeProto& shape_proto : operand_shapes_with_layout) {
           operand_shapes.emplace_back(shape_proto);
         }
-        instruction =
-            CreateCustomCall(shape, all_operands(), proto.custom_call_target(),
-                             operand_shapes, proto.backend_config());
+        if (proto.called_computation_ids_size() == 1) {
```

Comment:
Can you please add a TF_RET_CHECK that proto.called_computation_ids_size() <= 1 ? Or alternatively add support for more than 1 called computation? It seems somewhat bad if someone tries to use a custom call with layout constraint and with more than one called computation and the called computations are silently dropped.
Given that mlir_hlo_to_hlo seems to support only 1 called computation, my preference would be a TF_RET_CHECK.